swerve100/src/main/java/org/team100/frc2023/autonomous/HolonomicLQR.java

```diff
@@ -1,251 +1,221 @@
-        m_yManager.m_loop.correct(VecBuilder.fill(m_robotDrive.getPose().getY()));
-
-        m_xManager.m_loop.predict(0.020);
-        m_yManager.m_loop.predict(0.020);
-
-        // double nextXVoltage = m_xManager.m_loop.getU(0);
-        // double nextYVoltage = m_yManager.m_loop.getU(0);
-
-        xVolt.set(0.5 * m_lastXRef.getV() + m_lastXRef.getA());
-        yVolt.set(0.5 * m_lastYRef.getV() + m_lastYRef.getA());
-
-        // nextXVoltage + m_lastXRef.getA()
-        // nextYVoltage + m_lastYRef.getA()
-
-        double kV = 1.15;
-        double kA = 0.7;
-
-        Twist2d fieldRelativeTarget = new Twist2d(
-                ((kV * m_lastXRef.getV()) + (kA * m_lastXRef.getA())),
-                ((kV * m_lastYRef.getV()) + (kA * m_lastYRef.getA())),
-                thetaFF);
-
-        return fieldRelativeTarget;
-
-    }
-
-    public void reset(Pose2d currentPose) {
-        m_thetaController.reset(currentPose.getRotation().getRadians());
-        m_xManager.m_loop.reset(VecBuilder.fill(m_robotDrive.getPose().getX(), 0));
-        m_yManager.m_loop.reset(VecBuilder.fill(m_robotDrive.getPose().getY(), 0));
-
-        m_lastXRef = new MotionState(m_robotDrive.getPose().getX(), 0);
-
-        m_lastYRef = new MotionState(m_robotDrive.getPose().getY(), 0);
-
-    }
-
-    public void start() {
-        m_timer.restart();
-    }
-
-    public void updateProfile(double goalX, double endY, double maxVel, double maxAccel, double maxJerk) {
-        profileX = MotionProfileGenerator.generateSimpleMotionProfile(
-                new MotionState(m_robotDrive.getPose().getX(), 0),
-                new MotionState(goalX, 0),
-                maxVel,
-                maxAccel,
-                maxJerk);
-
-        profileY = MotionProfileGenerator.generateSimpleMotionProfile(
-                new MotionState(m_robotDrive.getPose().getY(), 0),
-                new MotionState(endY, 0),
-                maxVel,
-                maxAccel,
-                maxJerk);
-
-    }
-
-    /**
-     * Enables and disables the controller for troubleshooting problems. When
-     * calculate() is called on
-     * a disabled controller, only feedforward values are returned.
-     *
-     * @param enabled If the controller is enabled or not.
-     */
-    public void setEnabled(boolean enabled) {
-        m_enabled = enabled;
-    }
-
-    NetworkTableInstance inst = NetworkTableInstance.getDefault();
-    NetworkTable table = inst.getTable("Holonomic LQR");
-    DoublePublisher xOutPublisher = table.getDoubleTopic("X Output").publish();
-    DoublePublisher yOutPublisher = table.getDoubleTopic("Y Output").publish();
-    DoublePublisher xDesired = table.getDoubleTopic("X Desired").publish();
-    DoublePublisher yDesired = table.getDoubleTopic("Y Desired").publish();
-    DoublePublisher xVolt = table.getDoubleTopic("X Volt").publish();
-    DoublePublisher yVolt = table.getDoubleTopic("Y Volt").publish();
-}
+// package org.team100.frc2023.autonomous;
```

Comment:
you should be able to leave this alone if your stuff is completely separate, no?